src/components/AccountProvider.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Query } from 'react-apollo';
+import PropTypes from 'prop-types';
+import query from '../gql/account.graphql';
+
+export const AccountContext = React.createContext({
+  account: {
+    key: '1234',
+  },
+  brand: {
+    name: 'Firehouse.com',
+    logo: {
+      src: 'https://fortnight.imgix.net/ebm/5b476cd5e4f5510001847c65/fh_logo.png',
+    },
+  },
+});
+
+export const AccountProvider = props => (
+  <Query query={query}>
+    {({ loading, error, data }) => {
+      if (loading) {
+        return (
+          <p>Loading...</p>
```

Comment:
I'm assuming, ultimately, the pages will show two loading "screens?" One for the account and one for the story or story index?